.husky/check-commit-msg.js

```diff
@@ -7,8 +7,15 @@ function debug(debugMessage) {
 }
 
 const pattern = /^\w{2,4}-\d+\s/
-const message = process.argv[2]
-debug(`Test message "${message}" with pattern "${pattern}"`)
+const [message, currentBranch] = process.argv.slice(2)
```

Comment:
@quesabe can we instead create a general solution applicable to any rule? E.g. we could introduce a base sh script and call it before anything else in `.husky/commit-message` and other hooks.

Reply:
Not really. I would rather add a check right in the shell script [here](https://github.com/ottofeller/templates/blob/main/src/common/git/husky/assets/husky-shell-script-template). So that it's the same for all branches. I think it's achievable, because we can control the content of the script at synth time?

Reply:
Yes, moving the check to the shell script is possible. 
However two questions remain:
- Should I expand the `ignoreBranches` option to other rules?
- We currently have a possibility to add custom rules into `commit-msg ` hook. Thus there might be two or more rules inside the shell script. Should the branch check apply to all rules at once, or there should be a separate check for each rule?

Would it be a correct implementation?
```sh
#!/usr/bin/env sh
. "$(dirname -- "$0")/_/husky.sh"

# Check if branch is not ignored here (the list of branches is defined in `commitMsg` rule)
# 1. Execute this command only if branch is not listed in ignoreBranches
node .husky/check-commit-msg.js "$(head -1 $@)"

# Check if branch is not ignored (the list of branches is different and comes from `huskyCustomRule`)
# 2. Another rule goes here and executes only on non-ignored branches.
```

Reply:
@quesabe sorry not sure about the implementation, but the generic helper should check each rule in the shell script. Is this achievable?

Reply:
Got it. I see no obstacles - it's definitely doable.

Reply:
Great, thanks. Please keep e posted.

Reply:
Great, thanks. Please keep e posted.

Reply:
@gvidon I've reworked it to have the check in the shell script and it's now possible to apply the check to each of the rules separately. Also updated the description in this PR. Please take a look.

Reply:
Thanks, merged!